pkg/kubelet/network/dns/dns.go

```diff
@@ -397,6 +421,9 @@ func (c *Configurer) GetPodDNS(pod *v1.Pod) (*runtimeapi.DNSConfig, error) {
 			dnsConfig.Servers = []string{}
 			for _, ip := range clusterDNS {
 				dnsConfig.Servers = append(dnsConfig.Servers, ip.String())
+				nodeInfoMsg := fmt.Sprintf("For 1 TP X 1 RP verification - ClusterDNS IP : %q ", ip.String())
```

Comment:
please rephrase this message to get rid of 1x1 specific info

Reply:
Yes. Will remove "1 TP X 1 RP" specific info.